Tribler/Test/Core/Libtorrent/test_libtorrent_download_impl.py

```diff
@@ -140,3 +129,47 @@ def test_multifile_torrent(self):
         test_dict["a"] = "b"
         pstate.set("state", "engineresumedata", test_dict)
         return impl.network_create_engine_wrapper(pstate)
+
+    @deferred(timeout=10)
+    def test_save_resume(self):
+        """
+        testing call resume data alert
+        """
+        tdef = self.create_tdef()
+
+        impl = LibtorrentDownloadImpl(self.session, tdef)
+
+        def resume_ready(d=None):
+            """
+            check if resume data is ready
+            """
+            if not d:
+                d = defer.Deferred()
+
+            if not impl.pstate_for_restart:
+                reactor.callLater(2, resume_ready, d)
+            else:
+                basename = binascii.hexlify(tdef.get_infohash()) + '.state'
+                filename = os.path.join(self.session.get_downloads_pstate_dir(), basename)
+
+                engine_data = CallbackConfigParser()
+                engine_data.read_file(filename)
+
+                self.assertEqual(tdef.get_infohash(), engine_data.get('state', 'engineresumedata').get('info-hash'))
+
+                d.callback(None)
+
+            return d
+
+        def callback((dummy_1, dummy_2)):
```

Comment:
@devos50 oh, for sure, you cannot have two arguments named the same. we only need one here.
Sorry @ardhipoetra I didn't catch you mentioning the _two_ underscores. Just replace the whole thing with a single underscore. You are ignoring a tuple of two items instead of just ignoring the tuple altogether.